be/src/exec/schema_scanner.cpp

```diff
@@ -287,6 +287,12 @@ Status SchemaScanner::fill_dest_column_for_range(vectorized::Block* block, size_
                     reinterpret_cast<const char*>(&num), 0);
             break;
         }
+            // case TYPE_DECIMAL256: {
```

Comment:
Why delete this?

Reply:
This PR does not support create table with decimal256 type yet, will be supported in following PR.